libr/core/disasm.c

```diff
@@ -4191,11 +4217,12 @@ static void delete_last_comment(RDisasmState *ds) {
 		ds_begin_line (ds);
 		ds_setup_print_pre (ds, false, false);
 		if (!ds->linesright && ds->show_lines_bb && ds->line) {
-			char *refline = r_anal_reflines_str (ds->core, ds->at,
+			RAnalRefStr *refstr = r_anal_reflines_str (ds->core, ds->at,
 			                    ds->linesopts | R_ANAL_REFLINE_TYPE_MIDDLE_AFTER);
-			r_return_if_fail (refline);
-			r_cons_printf ("%s%s%s", COLOR (ds, color_flow),
-			               refline, COLOR_RESET (ds));
+			char *refline = refstr->str;
+			char *reflinecol = refstr->cols;
+			r_return_if_fail (refstr);
```

Comment:
wrong, remove this line, teh check must bne done before using it and we dont use asserts for runtime errors